<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1803</width>
    <height>1000</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="mouseTracking">
    <bool>true</bool>
   </property>
   <widget class="QLabel" name="status_bar">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>810</y>
      <width>941</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Awaiting input</string>
    </property>
   </widget>
   <widget class="QLabel" name="status_bar_label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>810</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Status Bar:</string>
    </property>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>30</y>
      <width>1502</width>
      <height>771</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="Hlayout">
     <property name="spacing">
      <number>0</number>
     </property>
     <property name="sizeConstraint">
      <enum>QLayout::SetNoConstraint</enum>
     </property>
     <item>
      <widget class="QScrollArea" name="scrollArea">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="widgetResizable">
        <bool>true</bool>
       </property>
       <widget class="QWidget" name="scrollAreaWidget_image_view">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>1498</width>
          <height>767</height>
         </rect>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Ignored" vsizetype="Ignored">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="mouseTracking">
         <bool>true</bool>
        </property>
        <widget class="QLabel" name="image_view">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>1500</width>
           <height>1500</height>
          </rect>
         </property>
         <property name="sizePolicy">
          <sizepolicy hsizetype="Ignored" vsizetype="Ignored">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="alignment">
          <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
         </property>
        </widget>
       </widget>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="image_resolution_label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>221</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>1. Set image rescale and contrast factors</string>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="image_cleanup_label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>130</y>
      <width>221</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>2. Cleanup image</string>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QSpinBox" name="small_objects_spinBox">
    <property name="geometry">
     <rect>
      <x>11</x>
      <y>160</y>
      <width>61</width>
      <height>22</height>
     </rect>
    </property>
    <property name="toolTipDuration">
     <number>-1</number>
    </property>
    <property name="maximum">
     <number>1000</number>
    </property>
    <property name="value">
     <number>45</number>
    </property>
   </widget>
   <widget class="QLabel" name="small_objects_label">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>160</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Small object pixel threshold (default 20)</string>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QSpinBox" name="large_objects_spinBox">
    <property name="geometry">
     <rect>
      <x>11</x>
      <y>200</y>
      <width>61</width>
      <height>22</height>
     </rect>
    </property>
    <property name="maximum">
     <number>10000000</number>
    </property>
    <property name="singleStep">
     <number>500</number>
    </property>
    <property name="value">
     <number>50000</number>
    </property>
   </widget>
   <widget class="QLabel" name="large_objects_label">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>200</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Large object pixel threshold (default 40000)</string>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="foreground_background_label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>280</y>
      <width>221</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>3. Set image background and foreground parameters</string>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QSpinBox" name="background_spinBox">
    <property name="geometry">
     <rect>
      <x>11</x>
      <y>330</y>
      <width>61</width>
      <height>22</height>
     </rect>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>20</number>
    </property>
    <property name="value">
     <number>3</number>
    </property>
   </widget>
   <widget class="QLabel" name="background_label">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>320</y>
      <width>161</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Background kernal size (default 3)</string>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="foreground_spinBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>370</y>
      <width>62</width>
      <height>22</height>
     </rect>
    </property>
    <property name="decimals">
     <number>1</number>
    </property>
    <property name="minimum">
     <double>0.100000000000000</double>
    </property>
    <property name="maximum">
     <double>1.000000000000000</double>
    </property>
    <property name="singleStep">
     <double>0.100000000000000</double>
    </property>
    <property name="value">
     <double>0.300000000000000</double>
    </property>
   </widget>
   <widget class="QLabel" name="foreground_label">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>370</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Foreground search range size (default 0.3)</string>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="run_edge_detection_label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>440</y>
      <width>221</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>4. Run seed edge detection algorithm</string>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="run_edge_detection_pushButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>490</y>
      <width>221</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Run seed edge detection algorithm</string>
    </property>
   </widget>
   <widget class="QLabel" name="calc_props_label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>530</y>
      <width>221</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>5. Calculate seed properties</string>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="calc_props_pushButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>560</y>
      <width>221</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Calculate seed properties</string>
    </property>
   </widget>
   <widget class="QPushButton" name="image_cleanup_pushButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>240</y>
      <width>221</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Cleanup image</string>
    </property>
   </widget>
   <widget class="QPushButton" name="image_resolution_pushButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>100</y>
      <width>211</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Confirm image scale</string>
    </property>
   </widget>
   <widget class="QPushButton" name="bg_fg_pushButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>410</y>
      <width>221</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Confirm parameters</string>
    </property>
   </widget>
   <widget class="QLabel" name="min_intensity_label">
    <property name="geometry">
     <rect>
      <x>79</x>
      <y>600</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Minimum display intensity</string>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QSpinBox" name="max_intensity_spinBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>630</y>
      <width>61</width>
      <height>22</height>
     </rect>
    </property>
    <property name="maximum">
     <number>255</number>
    </property>
    <property name="singleStep">
     <number>1</number>
    </property>
    <property name="value">
     <number>130</number>
    </property>
   </widget>
   <widget class="QLabel" name="max_intensity_label">
    <property name="geometry">
     <rect>
      <x>79</x>
      <y>630</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Maximum display intesity</string>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QSpinBox" name="min_intensity_spinBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>600</y>
      <width>61</width>
      <height>22</height>
     </rect>
    </property>
    <property name="maximum">
     <number>255</number>
    </property>
    <property name="value">
     <number>80</number>
    </property>
   </widget>
   <widget class="QLabel" name="viability_label">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>700</y>
      <width>161</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Viable seed cutoff</string>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QSpinBox" name="viability_spinBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>700</y>
      <width>61</width>
      <height>22</height>
     </rect>
    </property>
    <property name="maximum">
     <number>255</number>
    </property>
    <property name="value">
     <number>100</number>
    </property>
   </widget>
   <widget class="QPushButton" name="save_image_pushButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>740</y>
      <width>221</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Save current image</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="image_resolution_spinBox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>70</y>
      <width>61</width>
      <height>22</height>
     </rect>
    </property>
    <property name="maximum">
     <number>100</number>
    </property>
    <property name="value">
     <number>20</number>
    </property>
   </widget>
   <widget class="QPushButton" name="increase_contrast_pushButton">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>50</y>
      <width>71</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Increase
Contrast</string>
    </property>
   </widget>
   <widget class="QPushButton" name="decrease_contrast_pushButton">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>50</y>
      <width>71</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Decrease
Contrast</string>
    </property>
   </widget>
   <widget class="QPushButton" name="viability_pushButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>660</y>
      <width>221</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Classify viable seeds</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar_file">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1803</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen_DICOM_file"/>
    <addaction name="actionPrint_image"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>View</string>
    </property>
    <addaction name="actionZoom_in"/>
    <addaction name="actionZoom_out"/>
    <addaction name="actionZoom_reset"/>
    <addaction name="actionIncrease_Contrast"/>
    <addaction name="actionReduce_Contrast"/>
    <addaction name="actionReset_Contrast"/>
   </widget>
   <widget class="QMenu" name="menuPresets">
    <property name="title">
     <string>Presets</string>
    </property>
    <addaction name="actionAcacia_aulacophylla"/>
    <addaction name="actionAcacia_andrewsii"/>
    <addaction name="actionAcacia_aneura"/>
    <addaction name="actionAcacia_acuaria"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuView"/>
   <addaction name="menuPresets"/>
  </widget>
  <action name="actionOpen_DICOM_file">
   <property name="text">
    <string>Open DICOM file</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionPrint_image">
   <property name="text">
    <string>DNU - Print image</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+P</string>
   </property>
  </action>
  <action name="actionZoom_in">
   <property name="text">
    <string>Zoom in</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+=</string>
   </property>
  </action>
  <action name="actionZoom_out">
   <property name="text">
    <string>Zoom out</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+-</string>
   </property>
  </action>
  <action name="actionZoom_reset">
   <property name="text">
    <string>Reset zoom</string>
   </property>
  </action>
  <action name="actionIncrease_Contrast">
   <property name="text">
    <string>Increase Contrast</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+]</string>
   </property>
  </action>
  <action name="actionReduce_Contrast">
   <property name="text">
    <string>Reduce Contrast</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+[</string>
   </property>
  </action>
  <action name="actionReset_Contrast">
   <property name="text">
    <string>Reset Contrast</string>
   </property>
  </action>
  <action name="actionAcacia_andrewsii">
   <property name="text">
    <string>Acacia andrewsii</string>
   </property>
  </action>
  <action name="actionAcacia_aulacophylla">
   <property name="text">
    <string>Acacia aulacophylla</string>
   </property>
  </action>
  <action name="actionAcacia_aneura">
   <property name="text">
    <string>Acacia aneura</string>
   </property>
  </action>
  <action name="actionAcacia_acuaria">
   <property name="text">
    <string>Acacia acuaria</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
